setup.cfg

```diff
@@ -20,12 +20,14 @@ include_package_data = True
 package_dir =
     =src
 
-python_requires = >=3.7
+python_requires = >=3.9,<3.10
 
 install_requires =
     click>=8.0,<9.0
     importlib-metadata>=4.0
     python-dotenv>=0.19.2
+    starknet-devnet>=0.3.1
+    cairo-lang>=0.10.0
```

Comment:
I would add pytest requirement too, because we include a test example in the base project that depends on pytest, and having to install that manually from the user perspective seems not the best UX.

Reply:
> we include a test example in the base project that depends on pytest, and having to install that manually from the user perspective seems not the best UX.

Agreed!^ Though, `cairo-lang` already includes `pytest` and `pytest-asyncio` as deps...I don't know if there's a benefit in explicitly installing it. I suppose the same argument can be made for `cairo-lang` since devnet already has it as a dep

Reply:
I think that explicitly installing it helps us to define the version requirement for us, which in many cases could be not too important, but in certain occasions could be. 

**requests** is also a dependency we need to add btw.

Reply:
That's totally reasonable. Thank you 🙏 will update accordingly